### ISSUE TITLE
Fix ARIA for accordion

### ISSUE THREAD

**Issue by simonihmig** [kaliber5/ember-bootstrap#999]
* accordions have roles like `tab`, `tablist` and `tabpanel` stemming from Bootstrap 3 examples (https://getbootstrap.com/docs/3.4/javascript/#collapse-example-accordion). These should be removed.
* instead `aria-expanded` should be used, see https://getbootstrap.com/docs/4.4/components/collapse/#accessibility and https://www.w3.org/TR/wai-aria-practices/#wai-aria-roles-states-and-properties
